### ISSUE TITLE
documentation: mention that ParaView (Windows) and VisIt (all platforms) can't read BP5 files

### ISSUE THREAD

**Issue by BenWibking** [openPMD/openPMD-api#1576]
**What do you want to achieve, please describe.**

I'd like to open ADIOS2/BP5 files in VisIt and Paraview that were written with openPMD-api.

However, the current versions of Paraview (on Windows) and VisIt (on all platforms) cannot open BP5 files. See bug reports below:
* https://github.com/visit-dav/visit/issues/19176 (the bundled ADIOS2 version is too old to read BP5 files)
* https://discourse.paraview.org/t/cant-read-adios2-bp5-file/13535 (ADIOS2 doesn't suppport BP5 on Windows)

**Software Environment:**
Have you already installed openPMD-api?
If so, please tell us which *version* of openPMD-api your question is about:

 - version of openPMD-api: 0.15.2
 - installed openPMD-api via: conda-forge

**Comment by BenWibking** [comment]
This issue can be demonstrated with the test data attached to https://github.com/openPMD/openPMD-api/issues/1572.

**Comment by BenWibking** [comment]
Also, it looks like ADIOS2 itself can't be compiled with BP5 support on Windows.

**Comment by franzpoeschel** [comment]
FYI, there is planned support for BP5 on Windows in ADIOS2 v2.10, which is going to be released ahead of schedule for this reason.
Given that BP4 and BP5 both exist (and BP5 is not going to fully replace BP4), it is better to use the new filename extensions `.bp4` and `.bp5` now to avoid confusion, and also to potentially override the default file engine in ADIOS2.

Documentation-wise it's maybe generally a good time now to update some hints on ADIOS2 file engines and their use.

**Comment by BenWibking** [comment]
To clarify, will openPMD select the right engine automatically if I give it a filename with `.bp4` extension? That would certainly solve my current issues.

**Comment by franzpoeschel** [comment]
> To clarify, will openPMD select the right engine automatically if I give it a filename with `.bp4` extension? That would certainly solve my current issues.

Yep, `.bp4` and `.bp5` will explicitly select those ADIOS2 engines, while `.bp` will use whatever default ADIOS2 thinks should be used.
